.github/workflows/testing-and-deployment.yml

```diff
@@ -64,7 +64,11 @@ jobs:
           path: debug_images
 
   LinuxConda:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-20.04, ubuntu-24.04, ubuntu-24.04-arm, ubuntu-22.04-arm ]
```

Comment:
For this PR, it is better to narrowly focus on adding ARM, since there is no ubuntu-20.04-arm runner, we may want to first upgrade the runners images in a separate PR, so that we aren't mixing things here.

Reply:
perfect, fyi ubuntu20.04 will not run on March.

Reply:
See #7216